style.css

```diff
@@ -0,0 +1,123 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&family=Ubuntu:wght@300&display=swap");
+
+/**
+*! Color variables
+**/
+
+/**
+*! Removing default browser style
+**/
+* {
+  box-sizing: border-box;
+}
+
+* body {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  max-width: 23.75rem;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
```

Comment:
- [ ] Dear @tsheporamantso, as the previous code reviewer mentioned, the view should be responsive on all different mobile screen sizes, and, hence you set a ` max-width: 23.75rem;` your body is limited to only that size. Please consider removing this and all other width properties limiting your responsiveness. 

### Your design as of now. 
![Screenshot 2023-02-16 155041](https://user-images.githubusercontent.com/69077061/219370495-76e486d9-f480-40f5-8443-7a28fb961662.png)

Reply:
Thank you for the reply regarding the background images.  Can I please send you the PR on slack privately and check if the changes I've done are correct? If I remove the width as suggested the main section and hamburger are misaligned, but if I inspect with dimensions of (375 x 667) they get aligned.

Reply:
Sorry brother, I'm not on a computer right now. But I would say not to worry about the extra large mobile screens. Its obviouse the design will get a bit meesy as the screen grow. And in the future you will learn to use media query to adjust such displays. But for now make sure the display is responsive to that point.